### ISSUE TITLE
Suggestion: Sample with discrete/constant interpolation of scale 0↔1

### ISSUE THREAD

**Issue by donmccurdy** [KhronosGroup/glTF-Sample-Models#321]
In [a thread on the three.js forums](https://discourse.threejs.org/t/quill-vr-animation-to-glb/18668/19) we've come to the conclusion that a significant number of viewers cannot display constant interpolation of scale, particularly scale 0↔1.

- ✅ three.js
- ✅ PlayCanvas
- ❌ Babylon.js
- ❌ Khronos Sample Viewer
- ❌ Spark AR

The 'broken' three each show slightly different results — Spark AR doesn't appear to support constant interpolation at all, whereas Babylon.js plays the animation correctly _once_ and then reaches a broken state when trying to loop it.

It's a useful technique for 'baking' simulations as flipbook-style animations, showing each mesh for a single frame, but does depend on correct support for animating scale with constant interpolation.

**Comment by DrX3D** [comment]
This has been untouched for over a year. Is there anything to do here?

If there is no response, then this issue will be closed by 27 Nov.

**Comment by bghgary** [comment]
Babylon.js has already fixed this, but a sample model could still be useful.

**Comment by javagl** [comment]
By "constant", you mean `STEP` interpolation? 

There is a pending PR for the `InterpolationTest` at https://github.com/KhronosGroup/glTF-Sample-Assets/pull/26 . I could just change the values of the respective accessor...
```
  accessor10.setArray(new Float32Array([
    1, 1, 1, 
    0.5, 0.5, 0.5,  // Change to 0 0 0
    1, 1, 1, 
    0.5, 0.5, 0.5,  // Change to 0 0 0
    1, 1, 1
  ]));
```
if this is it...

**Comment by donmccurdy** [comment]
Yes, thanks – `STEP` interpolation with scale is what I had in mind.

**Comment by donmccurdy** [comment]
> An archive with the GLB and the code (hand-edited output of [gltfTransformifier](https://github.com/javagl/gltfTransformifier)) for generating it:

total aside, but that is a beautiful solution 🤯

**Comment by javagl** [comment]
OT: The name was inspired by [BCELifier](https://commons.apache.org/proper/commons-bcel/apidocs/org/apache/bcel/util/BCELifier.html). It's a **very** rough "proof-of-concept" for now, but I try to allocate some time to maybe turn it into something useful.

**Comment by javagl** [comment]
I have updated the PR at https://github.com/KhronosGroup/glTF-Sample-Assets/pull/26#issuecomment-1821000772 with the latest state that was posted here, where the scale interpolations go down to (0,0,0). When this PR is merged, then this issue can probably be closed.
